### PR TITLE
[New Rule] Suspicious Launchd Hidden Child Process

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/822

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/macos/persistence_defense_evasion_hidden_launch_agent_deamon_logonitem_process.toml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+[metadata]
+creation_date = "2020/01/07"
+maturity = "production"
+updated_date = "2020/01/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identify the execution of a launchd child process with a hidden file. An adversary can establish persistence by
+installing a new launch agent, daemons or logon item that executes at login.
```
Comment (bm11100) [inline rules/macos/persistence_defense_evasion_hidden_launch_agent_deamon_logonitem_process.toml:10]:
```suggestion
Identifies the execution of a launchd child process with a hidden file. An adversary can establish persistence by
installing a new logon item, launch agent, or daemon that executes upon login.
```

File: rules/macos/persistence_defense_evasion_hidden_launch_agent_deamon_logonitem_process.toml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Suspicious Launchd Hidden Child Process"
```
Comment (bm11100) [inline rules/macos/persistence_defense_evasion_hidden_launch_agent_deamon_logonitem_process.toml:16]:
```suggestion
name = "Suspicious Hidden Child Process of Launchd"
```

File: rules/macos/persistence_defense_evasion_hidden_launch_agent_deamon_logonitem_process.toml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+]
+risk_score = 47
+rule_id = "083fa162-e790-4d85-9aeb-4fea04188adb"
+severity = "medium"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Persistence", "Defense Evasion"]
+type = "query"
+
+query = '''
+event.category:process and event.type:(start or process_started) and
+ process.name : .* and process.parent.name : launchd
```
Comment (bm11100) [inline rules/macos/persistence_defense_evasion_hidden_launch_agent_deamon_logonitem_process.toml:30]:
```suggestion
 process.name:.* and process.parent.name:launchd
```

**Review by bm11100 — APPROVED** [review]

File: rules/macos/persistence_defense_evasion_hidden_launch_agent_deamon_logonitem_process.toml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+]
+risk_score = 47
+rule_id = "083fa162-e790-4d85-9aeb-4fea04188adb"
+severity = "medium"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Persistence", "Defense Evasion"]
+type = "query"
+
+query = '''
+event.category:process and event.type:(start or process_started) and
+ process.name:.* and process.parent.name:launchd
```
Comment (brokensound77) [inline rules/macos/persistence_defense_evasion_hidden_launch_agent_deamon_logonitem_process.toml:30]: ?

```suggestion
 process.name:.* and process.parent.executable:/sbin/launchd
```

**Review by brokensound77 — APPROVED** [review]
Added a suggestion, otherwise LGTM 👍

**Pull request merged**